std/container/array.d

```diff
@@ -539,11 +525,13 @@ Complexity: $(BIGOH 1)
     }
 
 /**
-Ensures sufficient capacity to accommodate $(D e) elements.
+Ensures sufficient capacity to accommodate $(D e) _elements.
+If $(D e < capacity), this method will not decrease the capacity.
 
 Postcondition: $(D capacity >= e)
 
-Complexity: $(BIGOH 1)
+Complexity: $(BIGOH e) if $(D e > capacity),
```

Comment:
There is such a thing as amortized complexity and this is clearly amortized O(1)